README.md

```diff
@@ -46,6 +46,30 @@ class Foo {
 }
 ```
 
+If you specify option `"scriptPath": true`:
+```js
+"plugins": [
+  ["annotate-console-log", {
+    "scriptPath": true
+  }]
+]
+```
+it'll add a short script path (and line number)
+```js
+console.log('apple')
+// =>
+console.log('[file.js:3]', 'foo()', 'apple')
+```
+Use `"fullScriptPath": true` to add more detailed script path info
```

Comment:
I don't like that `fullScriptPath` behaves different from `scriptPath`, i.e. one adds path as the first parameter of `console.log`, the second one adds it as an object to the end. I'd rather that `scriptPath` would always add a string path as the first parameter. Use it in combination with `scriptPathStyle` (ENUM `'relative' | 'fileName'`).